### PR TITLE
Added HasFlag Extension on enum for Framework 3.5

### PR DESCRIPTION
**Pull request by nazarenomanco**
@mauroservienti please check. 
This will be also useful on Radical.Windows

### CODE REVIEW

**Comment by mauroservienti** [comment]
@nazarenomanco you are pushing from an outdated branch that cannot be built, please update your local branches and push again.

File: src/Radical.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0
+# Visual Studio 14
+VisualStudioVersion = 14.0.24720.0
```
Comment (mauroservienti) [inline src/Radical.sln]: which Visual Studio version are you using?

**Comment by mauroservienti** [comment]
You don't necessarily need to create a test project specific for the .NET fx 3.5. Add the tests to the existing tests projects unless this has a meaning for 3.5 only. If this is intended to be used in all supported fx versions then add the changes (as linked files) to the other projects as well.

File: src/Radical.sln (hunk 1 above)
Comment (nazarenomanco) [inline src/Radical.sln]: Visual Studio 2015

**Comment by nazarenomanco** [comment]
This is an Extension Method implementation in fx 3.5 of an existing feature from fx 4.0.

**Pull request closed without merging**